documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| API family name                           | Quality by Design (QbD)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| API family owner                          | CableLabs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| API summary                               | API allowing 3rd party applications to:<br>- report appliction KPIs to network operator<br>- receive QbD score<br>- receive root cause and recommended corrective actions<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                         |
+|                                           | **API Scope**<br>This API would be part of the Home Device API Family. The QbD API allows a 3rd party application developer to create a session with a network operator. The application can then share KPIs such as:<br>- latency<br>- jitter<br>- packet loss<br>- bitrate<br>The network operator can monitors these KPIs and determine a QbD score. If the score ever drop below certain thresholds, an event can be triggered for rapid telemetry collection. The network operator can then determine root cause and suggest corrective actions either to the 3rd party app or to the network. |
```

Comment:
What about linking to the qos-profiles to show alignment with other CAMARA apis / KPIs?  https://github.com/camaraproject/QualityOnDemand/blob/main/code/API_definitions/qos-profiles.yaml

Reply:
good idea - I linked to the `qos-profiles`